source/extensions/common/aws/signer_impl.cc

```diff
@@ -24,8 +24,8 @@ void SignerImpl::sign(Http::RequestMessage& message, bool sign_body) {
 }
 
 void SignerImpl::sign(Http::RequestHeaderMap& headers) {
-  // S3 payloads require special treatment.
-  if (service_name_ == "s3") {
+  // S3, gracier, es payloads require special treatment.
```

Comment:
Paste the link to the docs explaining this requirement here?

Reply:
👍